### ISSUE TITLE
Allow aghosts to drag/drop ghosts onto bodies to SetMind + respawn as selected character.

### ISSUE THREAD

**Issue by keronshb** [space-wizards/space-station-14#21620]
## Description

Title explains it all.

**Comment by keronshb** [comment]
> Discord channels have taught me that it's really easy to accidentally drag and drop something

Fair point, added a request so it asks for confirmation before accepting.

**Comment by LordEclipse** [comment]
> > Discord channels have taught me that it's really easy to accidentally drag and drop something
> 
> Fair point, added a request so it asks for confirmation before accepting.

toggle+confirm
would be best way to go about it

**Comment by notquitehadouken** [comment]
admin accidentally forcing the soul of a greytider into someone else's body:

**Comment by ForestNoises** [comment]
What's meant by toggle in this issue? Like a togglable action (e.g. harm mode)?
